src/OptionSettlementEngine.sol

```diff
@@ -27,60 +27,44 @@ import "./TokenURIGenerator.sol";
 /// @author neodaoist
 contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
     /*//////////////////////////////////////////////////////////////
-    //  State variables - Public
+    //  Immutable/Constant - Private
     //////////////////////////////////////////////////////////////*/
 
-    uint8 public constant OPTION_ID_PADDING = 96;
+    // @dev The bit padding for option IDs
+    uint8 internal constant OPTION_ID_PADDING = 96;
 
-    uint96 private constant CLAIM_NUMBER_MASK = 0xFFFFFFFFFFFFFFFFFFFFFFFF;
+    // @dev The mask to mask out a claim number from a claimId
+    uint96 internal constant CLAIM_NUMBER_MASK = 0xFFFFFFFFFFFFFFFFFFFFFFFF;
+
+    /*//////////////////////////////////////////////////////////////
+    //  Immutable/Constant - Public
+    //////////////////////////////////////////////////////////////*/
 
     /// @notice The protocol fee
     uint8 public immutable feeBps = 5;
 
-    /// @notice Whether or not the protocol fee switch is enabled
-    bool public feeSwitch;
+    /*//////////////////////////////////////////////////////////////
+    //  State variables - Internal
+    //////////////////////////////////////////////////////////////*/
 
-    /// @notice The address fees accrue to
-    address public feeTo;
+    /// @notice Accessor for Option contract details
+    mapping(uint160 => OptionState) internal _optionStates;
```

Comment:
nit/not blocking: naming might be a tad opaque. can cover this in the natspec/doc review